### ISSUE TITLE
Cross Entropy Loss and its Derivations | Lee's Blog

### ISSUE THREAD

**Issue by AuthurWhywait** [AuthurWhywait/blog-comments#32]
https://authurwhywait.github.io/blog/2022/08/27/cross_entropy_loss/ 

 An introduction of Cross Entropy and some tricks
